examples/tests/requirements.txt

```diff
@@ -1,9 +1,10 @@
 # pytest 6.0 has linter-breaking changes
 # TODO(MLG-336): remove this requirement when tensorflow-macos is upgraded.
-protobuf<3.20; sys_platform == 'darwin' and platform_machine == 'arm64'
+protobuf>=3.20.3,<4.21; sys_platform == 'darwin' and platform_machine == 'arm64'
```

Comment:
this ends up being equivalent to `protobuf==3.20.3`, no?

Reply:
I knew you'd ask about this line. :-) I believe you're correct. I'm open to being totally strict here.